lib/ezutils/classes/ezextension.php

```diff
@@ -544,10 +544,10 @@ public static function getHandlerClass( ezpExtensionOptions $options )
             // we rely on the autoload system here
             if ( class_exists( $handler ) )
             {
-                // only use reflection if we have params to avoid exception on objects withouth constructor
-                if ( $handlerParams !== null && is_array( $handlerParams ) && count( $handlerParams ) > 0 )
+                $reflection = new ReflectionClass( $handler );
+                // only pass parameters if the class has a constructor
+                if ( $reflection->getConstructor() !== null && $handlerParams !== null && is_array( $handlerParams ) && count( $handlerParams ) > 0 )
                 {
-                    $reflection = new ReflectionClass( $handler );
```

Comment:
Since all the handlers will be called with the new injected parameter(s) with https://github.com/ezsystems/ezflow/pull/75 , it is a form of a BC break, so handlers could to be updated.
OTOH from a dev POV, why need to add an empty constructor to a handler that was working as intended?
This is mostly an issue in the 5.3 / 5.4 backports...

A warning seems like a safe choice, however since there are no constructors yet in existing handlers there may be a lot of warnings, updating handlers in flow is highly recommended too.